app/assets/js/main.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * Created by student on 17.11.16.
+ */
+
+$(document).ready(function () {
+
+    $(".site-spinner").fadeOut("slow", function () {
+        $(this).remove();
+    });
+
+
+    var change = $('.change');
+
+    change.each(function(){
+        var imgSrc = $(this).children().attr('src');
+        $(this).children("img").remove();
```

Comment:
You should decide what type of quote you want to use and use only it. You should not mix both types in your code.